oozie/src/main/java/org/apache/falcon/oozie/process/SparkProcessWorkflowBuilder.java

```diff
@@ -163,13 +164,16 @@ private void addOutputFeedsAsArgument(List<String> argList, Cluster cluster) thr
             return;
         }
 
-        for(Output output : entity.getOutputs().getOutputs()) {
+        int numOutputFeed = entity.getOutputs().getOutputs().size();
```

Comment:
I can understand you want to put input first and then output , but with in output or input ordering should be maintained right ?